packages/components/data-table-manager/src/data-table-settings/data-table-settings.js

```diff
@@ -3,6 +3,7 @@ import React, { useState } from 'react';
 import invariant from 'tiny-invariant';
 import { useIntl } from 'react-intl';
 import styled from '@emotion/styled';
+import AccessibleHiden from '@commercetools-uikit/accessible-hidden';
```

Comment:
```suggestion
import AccessibleHidden from '@commercetools-uikit/accessible-hidden';
```

This confused me when using CTRL+F 😛